src/lib/components/Swap/WrapErrorText.tsx

```diff
@@ -0,0 +1,22 @@
+import { Trans } from '@lingui/macro'
+import { WrapInputError } from 'lib/hooks/swap/useWrapCallback'
+import useNativeCurrency from 'lib/hooks/useNativeCurrency'
+
+export function WrapErrorText({ wrapInputError }: { wrapInputError: WrapInputError }) {
```

Comment:
I think this file came in from a different PR and shouldn't be here?

Reply:
gah thanks